pyFV3/wrappers/__init__.py

```diff
@@ -1,10 +0,0 @@
-from .geos_wrapper import GeosDycoreWrapper, MemorySpace, StencilBackendCompilerOverride
```

Comment:
I like geos_wrapper in here, personally, but it's not important until/unless we have other wrappers supported

Reply:
They have been brought back into that `__init__`